src/compiler/config/config-utils.ts

```diff
@@ -21,64 +44,31 @@ export const setBooleanConfig = (config: any, configName: string, flagName: stri
     config[userConfigName] = !!config[userConfigName]();
   }
 
-  if (typeof config[userConfigName] === 'boolean') {
+  if (isBoolean(config[userConfigName])) {
     config[configName] = config[userConfigName];
   } else {
     config[configName] = defaultValue;
   }
 };
 
-export const setNumberConfig = (config: any, configName: string, _flagName: string, defaultValue: number) => {
```

Comment:
I removed a few functions here because they were commited in 2018 and I couldn't find any uses of them

Reply:
excellent! deleting code is the best kind of refactoring